source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,268 @@
+=======
+Indexes
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Indexes are data structures that support the efficient execution of queries in MongoDB. They contain copies of parts of
+the data in documents to make queries more efficient.
+
+Without indexes, MongoDB must scan *every* document in a collection to find the documents that match each query. These
+collection scans are slow and can negatively affect the performance of your application. By using an index to limit the
+number of documents MongoDB scans, queries can be more efficient and return faster.
+
+
+Query Coverage and Performance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you execute a query against MongoDB, your query can include three parts:
+
+- query criteria that specify field(s) and value(s) you are looking for
+- options that affect the query's execution (e.g. read concern)
```

Comment:
This isn't really relevant to indexes.  Consider removing.

Reply:
I think it's valuable to frame indexes in how they are used, and this is just a small reminder to readers what a query operation includes. Thoughts?

Reply:
I still find it confusing.  Perhaps if you want to include it, separate it out into a section that makes it clear that it's not relevant to index use.

On a related note, this whole section seems geared towards the find command, though indexes are using for aggregation and updates as well.